### ISSUE TITLE
1m high-res and NIR band

### ISSUE THREAD

**Issue by VaasuDevanS** [allenai/satlas-super-resolution#10]
Thanks very much to the authors for making this work in the public domain. I have few questions.

1.) NAIP imagery is available in 1m but why the super resolution result is only 2.5m? Are there any technical challenges in training the model to generate 1m image?

2.) NAIP imagery also includes the NIR band. I am curious why the NIR band was not included in the super resolution process. For example, using the `Sentinel-2 10m (R, G, B, NIR)` as model input to get the high resolution `2.5m (R, G, B, NIR)` instead of just using the RGB bands.

3.) I looked at the `val_set` (as it was easier to download and check) and only could see the `rgb` as png file inside the NAIP folder. Is there any chance to make the NIR band from NAIP available for the training data?

Thanks again!

**Comment by piperwolters** [comment]
Hi, thank you for your interest in this work!

1) We found that training models to upsample Sentinel-2 imagery by 4x to get to 2.5m/px imagery (downsampled NAIP) worked the best as far as balancing realistic outputs and semantic accuracy. We tried training 8x and 16x models, but found that the models weren't as accurate in producing buildings where there should be buildings and things like that. Not to say it won't be done someday, but we focused on 4x.
2) We did not include the NIR band because we wanted to focus on RGB. Our intention with super-resolution was to produce global high-res imagery that could be used as an additional source for annotation, so for visualization purposes, using just RGB was better. 
3) It could be possible for you to download the NIR band for all of the training data, using the filepaths for each NAIP image, but we probably won't be able to do that in the near-term.
